packages/cli/src/commonlib/codeFlowLogin.ts

```diff
@@ -378,12 +377,12 @@ export class CodeFlowLogin {
   }
 }
 
-function sendFile(
+async function sendFile(
   res: http.ServerResponse,
   filepath: string,
   contentType: string,
   accountName: string
-) {
+): Promise<void> {
   fs.readFile(filepath, (err, body) => {
```

Comment:
`await fs.readFile` or call `resolve` in callback, otherwise it's not a real async function...

Reply:
Fixed.